examples/dotnet/PdxAutoSerializer/Program.cs

```diff
@@ -28,6 +28,10 @@ public static void Main(string[] args)
                 .Set("log-level", "none");
             var cache = cacheFactory.Create();
 
+            Console.WriteLine("Registering for reflection-based auto serialization");
```

Comment:
Why is the example changing in this PR?

Reply:
It wasn't fixed in another commit, so the examples were failing to build when testing this